<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDlg</class>
 <widget class="QDialog" name="MainDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainDlg</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="CTitleBar" name="titleBar" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnMenu">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>287</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>China divination</string>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>287</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnMenuMin">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnMenuClose">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <layout class="QGridLayout" name="gridLayout">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_8">
                    <property name="font">
                     <font>
                      <family>楷体</family>
                      <pointsize>14</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>起卦时间：</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="lineEdit">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_10">
                    <property name="font">
                     <font>
                      <family>楷体</family>
                      <pointsize>14</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>占事类别：</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QComboBox" name="comboBox_7"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <spacer name="verticalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;使用铜钱进行爻卦，将每次爻卦结果填写在下面&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QWidget" name="widget" native="true">
                <property name="minimumSize">
                 <size>
                  <width>400</width>
                  <height>300</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QWidget{border:1px solid #000000;
border-radius:2px;
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,  stop:0 #FFFFF1,  stop: 1 #FFFFFF);
}</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <layout class="QGridLayout" name="gridLayout_2">
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_7">
                     <property name="font">
                      <font>
                       <family>华文楷体</family>
                       <pointsize>14</pointsize>
                      </font>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::NoFrame</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Plain</enum>
                     </property>
                     <property name="midLineWidth">
                      <number>0</number>
                     </property>
                     <property name="text">
                      <string>六爻：</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                     <property name="margin">
                      <number>10</number>
                     </property>
                    </widget>
                   </item>
                   <item row="3" column="1">
                    <widget class="QComboBox" name="comboBox_4">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="font">
                      <font>
                       <family>华文楷体</family>
                       <pointsize>15</pointsize>
                       <weight>50</weight>
                       <bold>false</bold>
                       <kerning>true</kerning>
                      </font>
                     </property>
                     <item>
                      <property name="text">
                       <string>少阳▅▅▅▅▅</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>少阴▅▅    ▅▅</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>老阳▅▅▅▅▅O</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>老阴▅▅    ▅▅X</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                   <item row="2" column="1">
                    <widget class="QComboBox" name="comboBox_3">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="font">
                      <font>
                       <family>华文楷体</family>
                       <pointsize>15</pointsize>
                       <weight>50</weight>
                       <bold>false</bold>
                       <kerning>true</kerning>
                      </font>
                     </property>
                     <item>
                      <property name="text">
                       <string>少阳▅▅▅▅▅</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>少阴▅▅    ▅▅</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>老阳▅▅▅▅▅O</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>老阴▅▅    ▅▅X</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                   <item row="5" column="0">
                    <widget class="QLabel" name="label">
                     <property name="font">
                      <font>
                       <family>华文楷体</family>
                       <pointsize>14</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>初爻：</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                     <property name="margin">
                      <number>10</number>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="label_6">
                     <property name="font">
                      <font>
                       <family>华文楷体</family>
                       <pointsize>14</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>五爻：</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                     <property name="margin">
                      <number>10</number>
                     </property>
                    </widget>
                   </item>
                   <item row="4" column="0">
                    <widget class="QLabel" name="label_3">
                     <property name="font">
                      <font>
                       <family>华文楷体</family>
                       <pointsize>14</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>二爻：</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                     <property name="margin">
                      <number>10</number>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="0">
                    <widget class="QLabel" name="label_5">
                     <property name="font">
                      <font>
                       <family>华文楷体</family>
                       <pointsize>14</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>四爻：</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                     <property name="margin">
                      <number>10</number>
                     </property>
                    </widget>
                   </item>
                   <item row="5" column="1">
                    <widget class="QComboBox" name="comboBox_6">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="font">
                      <font>
                       <family>华文楷体</family>
                       <pointsize>15</pointsize>
                       <weight>50</weight>
                       <bold>false</bold>
                       <kerning>true</kerning>
                      </font>
                     </property>
                     <item>
                      <property name="text">
                       <string>少阳▅▅▅▅▅</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>少阴▅▅    ▅▅</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>老阳▅▅▅▅▅O</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>老阴▅▅    ▅▅X</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QComboBox" name="comboBox">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="font">
                      <font>
                       <family>华文楷体</family>
                       <pointsize>15</pointsize>
                       <weight>50</weight>
                       <bold>false</bold>
                       <kerning>true</kerning>
                      </font>
                     </property>
                     <item>
                      <property name="text">
                       <string>少阳▅▅▅▅▅</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>少阴▅▅    ▅▅</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>老阳▅▅▅▅▅O</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>老阴▅▅    ▅▅X</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                   <item row="3" column="0">
                    <widget class="QLabel" name="label_4">
                     <property name="font">
                      <font>
                       <family>华文楷体</family>
                       <pointsize>14</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>三爻：</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                     <property name="margin">
                      <number>10</number>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QComboBox" name="comboBox_2">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="font">
                      <font>
                       <family>华文楷体</family>
                       <pointsize>15</pointsize>
                       <weight>50</weight>
                       <bold>false</bold>
                       <kerning>true</kerning>
                      </font>
                     </property>
                     <item>
                      <property name="text">
                       <string>少阳▅▅▅▅▅</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>少阴▅▅    ▅▅</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>老阳▅▅▅▅▅O</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>老阴▅▅    ▅▅X</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                   <item row="4" column="1">
                    <widget class="QComboBox" name="comboBox_5">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="font">
                      <font>
                       <family>华文楷体</family>
                       <pointsize>15</pointsize>
                       <weight>50</weight>
                       <bold>false</bold>
                       <kerning>true</kerning>
                      </font>
                     </property>
                     <item>
                      <property name="text">
                       <string>少阳▅▅▅▅▅</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>少阴▅▅    ▅▅</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>老阳▅▅▅▅▅O</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>老阴▅▅    ▅▅X</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>隶书</family>
                <pointsize>16</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>开始起卦</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTitleBar</class>
   <extends>QWidget</extends>
   <header location="global">ctitlebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
